OpenAI/Packages/com.openai.unity/Runtime/Threads/CreateMessageRequest.cs

```diff
@@ -1,30 +1,44 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using UnityEngine.Scripting;
 
 namespace OpenAI.Threads
 {
-    [Preserve]
+    [Obsolete("use Thread.Message instead.")]
```

Comment:
@StephenHodgson I began porting over TextContent support from the DotNet version, and noticed this class is marked obsolete in Unity but not in DotNot. 
When serializing the Unity Theads.Message class however, it fails, as the API expects just a Content string (not a text object). So it appears that the CreateMessageRequest would still be useful as the API message send/receive are seemingly different.

As brief guidance here, do you expect this class to still be re-added to support this case?

Reply:
it is also obsolete in the DotNet version. The data structures are exactly the same so I'm removing duplicate class.